src/microbe_stage/OrganismStatisticsPanel.cs

```diff
@@ -198,9 +208,11 @@ public void UpdateStatVisibility()
         digestionEfficiencyLabel.Visible = ShowDigestionEfficiencyStat;
         digestionStatsSeparator.Visible = ShowDigestionSpeedStat || ShowDigestionEfficiencyStat;
 
+        atpBalanceBarContainer.Visible = ShowATPBalanceBars;
+
         ammoniaCostLabel.Visible = ShowOrganellesCostStat;
         phosphatesCostLabel.Visible = ShowOrganellesCostStat;
-        digestionStatsSeparator.Visible = ShowOrganellesCostStat;
+        organellesCostsSeparator.Visible = ShowOrganellesCostStat;
```

Comment:
When merging master I noticed there was probably a typo in @Patryk26g's PR, which I've corrected here.